lib/class-factory.js

```diff
@@ -15,6 +15,7 @@ const {
   getArrayType,
   makeJniObjectTypeName
 } = require('./types');
+const memoize = require('./memoize');
```

Comment:
Imports should be sorted alphabetically by module name.

Reply:
Fixed, should not have been there